core/neat/functions/_retrieve-neat-settings.scss

```diff
@@ -13,6 +13,14 @@
 /// @access private
 
 @function _retrieve-neat-setting($grid, $setting) {
+  $_neat-grid-defaults: map-merge((
```

Comment:
With this added functionality, can we expand upon the description for this function on Line 2?